### PR TITLE
Update DEVELOPERS.md

### PR DESCRIPTION
**Pull request by gargarchit**
I found some Improper word choice as well as Improper formatting for that I have made changes and corrected that so, now It will be easier for everyone to read.
Please merge this
https://github.com/accordproject/techdocs/issues/226

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -147,10 +147,10 @@ Formatting for the body is displayed in this example:
 
 ### Changes
 - Change one
-  - Subchange one
-  - Subchange two
+  - Sub change one
+  - Sub change two
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ We have very precise rules over how our git commit messages can be formatted.  T
 readable messages** that are easy to follow when looking through the **project history** and **git logs**.  
 But also, we use the git commit messages to **generate the Markdown-Transform change log**.
 
-The commit message formatting can be added using a version of typical git workflow.
+The commit message formatting can be added using a version of the typical git workflow.
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ npm install
 
 ### Keeping In Sync
 
-It is good practice to always keep your `origin/master` in sync with `upstream/master`. You don’t have to, but it makes your life easier. Do your work in branches of your fork, and periodically sync up your `master` with the `master` of `upstream` as follows. You should definitely do this before creating a pull request.
+It is good practice to always keep your `origin/master` in sync with `upstream/master`. You don’t have to, but it makes your life easier. Do your work in branches of your fork, and periodically sync up your `master` with the `master` of `upstream` as follows. You should do this before creating a pull request.
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

File: DEVELOPERS.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ machine:
 * [Git][git]: The [Github Guide to Installing Git][git-setup] is a good source of information.
 
 * [Node.js v10.16.0 'Dubnium' (LTS)][node]: We use Node to generate the documentation, run a
-  development web server, run tests, and generate distributable files. Depending on your system,
+  development of web server, run tests and generate distributable files. Depending on your system,
```
Comment (irmerk) [inline DEVELOPERS.md]: This can be removed.

**Comment by jeromesimeon** [comment]
No activity on this. I'm closing the PR.

**Pull request closed without merging**